app.hypermedia.testing.dsl.cli/pom.xml

```diff
@@ -12,6 +12,12 @@
         <maven.compiler.target>1.8</maven.compiler.target>
     </properties>
 
+    <parent>
+        <groupId>app.hypermedia.testing.dsl</groupId>
+        <artifactId>app.hypermedia.testing.dsl.parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
```

Comment:
Why is this needed?

Reply:
This is set in every other module but this one for some reason. Without this I had an error saying it didn't found the `jib:build` target in this module

Reply:
> for some reason

Hah, quite simple. The others were built from generator and this I half-handcrafted :)